33 - vip area location columns

chicharr opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #33

- Before: https://main--realmadrid--hlxsites.hlx.page//area-vip/partido-vip/silver-club-127
- After: https://33-vip-location-columns--realmadrid--hlxsites.hlx.page/area-vip/partido-vip/silver-club-127

vdua commented:
There is some minor color and margin issue in both the sites, but other than that looks good to me

Desktop

| **Original** | **Franklin** |
|---|---|
|<img width="755" alt="image" src="https://user-images.githubusercontent.com/2097597/236989020-ca82e365-b8ab-4cd9-b4a0-bd26250e7e14.png">|<img width="692" alt="image" src="https://user-images.githubusercontent.com/2097597/236989062-3fc50349-c35f-44b6-b3c9-919d73fa54d7.png">|

Mobile

In Mobile the border of the heading covers the full width


| **Original** | **Franklin** |
|---|---|
|<img width="719" alt="image" src="https://user-images.githubusercontent.com/2097597/236989381-2d1385f8-92a0-4159-9a99-10cb320b7464.png">|<img width="717" alt="image" src="https://user-images.githubusercontent.com/2097597/236989352-060db48d-afd4-4754-918a-5d074bfb3392.png">|

chicharr commented:
> There is some minor color and margin issue in both the sites, but other than that looks good to me
I fixed the margin. @vdua  please be aware that we agreed with RM to change the fonts in order to unify them with Tour website.

> In Mobile the border of the heading covers the full width
Fixed
